src/iosource/pcap/Source.cc

```diff
@@ -186,6 +186,13 @@ void PcapSource::OpenOffline()
 		InternalError("OS does not support selectable pcap fd");
 
 	props.link_type = pcap_datalink(pd);
+#ifdef DONT_HAVE_LIBPCAP_DLT_LINUX_SLL2
+        if ( props.link_type == 276 )
+            {
+		Error("PCAP file is using DLT_LINUX_SLL2 which is unsupported by your libpcap version");
```

Comment:
Worth an InternalError that also aborts execution?

Reply:
@awelzel yeah `InternalError` was the first thing I did but then I grepped through the codebase and it seems this is used for errors that are truly internal (e.g., consequence of a bug), while this feels more like an "external error" :grin: (i.e., user is asking us to do something unsupported on this system). Please advise if I have this right...